### ISSUE TITLE
Please remove deprecation from Actions.pause()

### ISSUE THREAD

**Issue by mmichaelis** [SeleniumHQ/selenium#1071]
I understand the bad design mentioned in the deprecation note in `Actions.pause()`. It's from one perspective just as bad as using `Thread.sleep()`. So I agree that usage should be considered possibly harmful.

Nevertheless it was the only option for us to perform stable drag-and-drop in a Rich-Web-UI:
- We needed to pause on the source after click-and-hold.
- We needed to cycle a little on the target and then pause before actually releasing the mouse button.

Our helper code for drag-and-drop which has ripened for years now and has proven to be very stable is:

``` java
public void dragFromTo(final WebElement source, final WebElement target, final int offsetX, final int offsetY) {
    bypassIEScrollIntoViewBugWorkaround(target);

    final Actions actionProvider = createActions(source);
    // Pausing - bad practice: Yes, pausing is bad practice but unless you know states to wait for
    // there is no other chance then giving the UI some time to e. g. activate a drop-zone.
    Actions startDrag = actionProvider.clickAndHold(source).pause(500);
    Actions moveTo = startDrag.moveToElement(target).moveByOffset(offsetX, offsetY);
    Actions waitSomeTime = moveTo.moveByOffset(1, 1).moveByOffset(-1, -1).pause(500);
    Actions dropElement = waitSomeTime.release();
    final Action completeAction = dropElement.build();
    try {
      completeAction.perform();
    } catch (WebDriverException e) {
      reportOnException(e, source, target);
      throw e;
    }
}
```

The pause really was essential and fixed about 80% of our flaky drag and drop tests.

I am now about to add a SuppressWarnings to the code... but I think it would be better to just turn the warning into a normal JavaDoc.

**Comment by Mobrockers** [comment]
Agreed. I need the pause on a context menu with submenu's to wait a bit until the submenu has been generated and opened before proceeding with the action chain, but using a webdriverwait does not seem to work in this case, and pause() works perfectly. Since there seems to be no suggested alternative, this needs to either be un-deprecated or an alternative needs to be suggested in the docs if an alternative already exists.

**Comment by barancev** [comment]
Closed by 7b81ced5464799587fc582fb5e5bb8560098fab2
